python/tvm/topi/vision/ssd/multibox.py

```diff
@@ -238,7 +238,8 @@ def hybrid_multibox_transform_loc(cls_prob, loc_pred, anchor, clip, threshold, v
                 out_loc[i, valid_count[i], 4] = out_coord[2]
                 out_loc[i, valid_count[i], 5] = out_coord[3]
                 valid_count[i] += 1
-
+        for j in range(valid_count[i], num_anchors):
+            out_loc[i, j, 0] = -1.0
```

Comment:
For the initialization, `out_loc[i, j, 1~5]` are also not initialized right? cc @kevinthesun

Reply:
The class id is the most important part, usually if we set it to -1 as background, values on out_loc[i, j, 1~5] then will be ignored, so for a minimum change to fix, this should be enough.